service/sg_accel_service_uninstall.sh

```diff
@@ -4,16 +4,18 @@
 OS=""
 VER=""
 SERVICE_NAME="sg_accel"
-SRCCFGDIR=../examples
+SCRIPT_DIR="$( cd "$( dirname "$0" )" > /dev/null && pwd )"
```

Comment:
Looks like the identical function as above.  How painful would it be to refactor this out into a common function that was included by both .sh files?